src/ur3e_hande_moveit_config/README.md

```diff
@@ -0,0 +1,6 @@
+## ur3e_hande_moveit_config
+
+This package defines configuration parameters of the combined robotic system of hand-e gripper and ur3e robot arm. 
+Included files are autogenerated through the "MoveIt Setup Assistant" program and modified according to the need afterwards. 
+
+
```

Comment:
Feel free to add some more detailed notes from the overleaf. It doesn't have to be completely pedantic, but should make it feasible for a newcomer to work with.

Reply:
Contents from the overleaf were added. 
https://github.com/maffettone/erobs/blob/b5e2b1b799c5b040c2a1f2375dd0bee01f042222/src/ur3e_hande_moveit_config/README.md?plain=1#L6-L17